File: components_page/components/dropdown/dark.jl
using DashBootstrapComponents

dropdown = dbc_dropdownmenu(
    label = "Menu",
    menu_variant = "dark",
    children = [
        dbc_dropdownmenuitem("Item 1"),
        dbc_dropdownmenuitem("Item 2"),
        dbc_dropdownmenuitem("Item 3"),
    ],
)

File: docs/components_page/components/dropdown/dark.py
import dash_bootstrap_components as dbc

dropdown = dbc.DropdownMenu(
    label="Menu",
    menu_variant="dark",
    children=[
        dbc.DropdownMenuItem("Item 1"),
        dbc.DropdownMenuItem("Item 2"),
        dbc.DropdownMenuItem("Item 3"),
    ],
)
